test/integration/mysql-ha/mysql_ha_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql_ha
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMySqlHaModule(t *testing.T) {
+
+	const databaseVersion = "MYSQL_5_7"
+	const backendType = "SECOND_GEN"
+	const state = "RUNNABLE"
+	const region = "us-central1"
+
+	const activationPolicy = "ALWAYS"
+	const dataDiskSizeGb = int64(10)
+	const kind = "sql#settings"
+	const pricingPlan = "PER_USE"
+	const replicationType = "SYNCHRONOUS"
+	const storageAutoResize = true
+	const storageAutoResizeLimit = int64(0)
+	const tier = "db-n1-standard-1"
+
+	const locationPreferenceKind = "sql#locationPreference"
+
+	const maintenanceWindowKind = "sql#maintenanceWindow"
+	const maintenanceWindowDay = int64(7)
+	const maintenanceWindowHour = int64(12)
+	const maintenanceWindowUpdateTrack = "stable"
+
+	const databaseFlags = `{
+        "name": "long_query_time",
+        "value": "1"
+      }`
+
+	const binaryLogEnabled = true
+	const backupConfigurationEnabled = true
+	const backupConfigurationKind = "sql#backupConfiguration"
+	const backupConfigurationStartTime = "20:55"
+	const backupConfigurationRetainedBackups = int64(365)
+	const backupConfigurationRetentionUnit = "COUNT"
+
+	mySql := tft.NewTFBlueprintTest(t)
+
+	mySql.DefineVerify(func(assert *assert.Assertions) {
+		mySql.DefaultVerify(assert)
+
+		instaceNames := []string{mySql.GetStringOutput("name")}
+		op := gcloud.Run(t, fmt.Sprintf("sql instances describe %s --project %s", instaceNames[0], mySql.GetStringOutput("project_id")))
+		assert.Equal(len(op.Get("replicaNames").Array()), 3, "Expected 3 replicas")
+		instaceNames = append(instaceNames, op.Get("replicaNames").Array()[0].String())
+		instaceNames = append(instaceNames, op.Get("replicaNames").Array()[1].String())
+		instaceNames = append(instaceNames, op.Get("replicaNames").Array()[2].String())
+
```

Comment:
You can use https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test@v0.0.0-20220210045134-f3d60ff938cb/pkg/utils#GetResultStrSlice
```suggestion
		instaceNames = append(instaceNames, utils.GetResultStrSlice(t, op.Get("replicaNames")))

```